### PR TITLE
CLDR-16122 v43 BRS -z… FINAL_TESTING

### PR DESCRIPTION
**Pull request by srl295**
CLDR-16122

exact redo of #2657  + 1 fix

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/main/nl_BE.xml [hunk 1]
```diff
@@ -216,7 +216,6 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 			<nameField type="given" draft="provisional">Juliette</nameField>
 			<nameField type="given-informal" draft="provisional">Juli</nameField>
 			<nameField type="given2" draft="provisional">↑↑↑</nameField>
-			<nameField type="surname" draft="provisional">Van den Berg</nameField>
```
Comment (srl295) [inline common/main/nl_BE.xml:219]: @macchiati  is this correct?

**Comment by srl295** [comment]
console check with -z FINAL_TESTING passes for me locally.

File: common/main/nl_BE.xml (hunk 1 above)
Comment (srl295) [reply common/main/nl_BE.xml:219]: the err message said surname should not be there

Comment (pedberg-icu) [reply common/main/nl_BE.xml:219]: Yes, that is correct. I was just creating a PR to do the same thing.

Comment (pedberg-icu) [reply common/main/nl_BE.xml:219]: It is redundant since we have surname-prefix and surname-core

**Review by pedberg-icu — APPROVED** [review]

**Comment by pedberg-icu** [comment]
I am going to go ahead and merge since I have other changes I want to start on...

File: common/main/nl_BE.xml (hunk 1 above)
Comment (srl295) [reply common/main/nl_BE.xml:219]: @pedberg-icu thanks.

**Pull request merged**